testnet/stacks-node/conf/mainnet-mockminer-conf.toml

```diff
@@ -1,17 +1,13 @@
 [node]
-# working_dir = "/dir/to/save/chainstate"
+# working_dir = "/dir/to/save/chainstate" # defaults to: /tmp/stacks-node-[0-9]*
 rpc_bind = "0.0.0.0:20443"
 p2p_bind = "0.0.0.0:20444"
 miner = true
 mock_mining = true
 bootstrap_node = "02196f005965cebe6ddc3901b7b1cc1aa7a88f305bb8c5893456b8f9a605923893@seed.mainnet.hiro.so:20444,02539449ad94e6e6392d8c1deb2b4e61f80ae2a18964349bc14336d8b903c46a8c@cet.stacksnodes.org:20444,02ececc8ce79b8adf813f13a0255f8ae58d4357309ba0cedd523d9f1a306fcfb79@sgt.stacksnodes.org:20444,0303144ba518fe7a0fb56a8a7d488f950307a4330f146e1e1458fc63fb33defe96@est.stacksnodes.org:20444"
+prometheus_bind = "0.0.0.0:9153" 
 
 [burnchain]
 chain = "bitcoin"
 mode = "mainnet"
-peer_host = "bitcoind.stacks.co"
-username = "blockstack"
-password = "blockstacksystem"
-rpc_port = 8332
-peer_port = 8333
-burn_fee_cap = 1
+peer_host = "bitcoin.hiro.so"
```

Comment:
Why is this changing? My very strong preference is to _not_ use Hiro nodes in default configs. The Foundation runs many public Bitcoin nodes right, can you use one of them instead?

Reply:
that line was only changed for correctness from this: https://github.com/stacks-network/stacks-core/blob/develop/testnet/stacks-node/conf/mainnet-mockminer-conf.toml#L12

which referred to `bitcoind.stacks.co`

Reply:
this is set to localhost now